### ISSUE TITLE
EnforcedStyleForEmptyBraces in autogenerated config

### ISSUE THREAD

**Issue by herwinw** [bbatsov/rubocop#4887]
I have a very large legacy codebase, with most block calls like this:

```ruby
i_want_a_block{ foo }
```

Using `--autogenerate` results in the following todo-file:

```yaml
# Offense count: 2195
# Cop supports --auto-correct.
# Configuration parameters: EnforcedStyle, SupportedStyles, SupportedStylesForEmptyBraces.
# SupportedStyles: space, no_space
# SupportedStylesForEmptyBraces: space, no_space
Layout/SpaceBeforeBlockBraces:
  EnforcedStyleForEmptyBraces: no_space
```

But EnforcedStyleForEmptyBraces is not a valid configuration option, so running Rubocop again results in:

```
Warning: unrecognized parameter Layout/SpaceBeforeBlockBraces:EnforcedStyleForEmptyBraces found in .rubocop_todo.yml
```

Afterwards it shows a bunch of errors, because all the block calls are still invalid.

--------

## Expected behavior

Add `Enabled: false` in the todo file

## Actual behavior

See above

## Steps to reproduce the problem

I'm not at liberty to share the code base, I guess you really need a lot of code for it to behave this way.

## RuboCop version

```
$ rubocop -V
0.51.0 (using Parser 2.4.0.0, running on ruby 2.1.5 i386-linux-gnu)
```
